src/lib.rs

```diff
@@ -47,7 +47,7 @@ mod interrupt_ext;
 
 pub use crate::_generated::{peripherals, Peripherals};
 
-#[cfg(any(systick_rv2, systick_rv3))]
+#[cfg(any(systick_rv2, systick_rv3, systick_rv4))]
```

Comment:
Better make it `not(time-driver-systick)`. `time-driver-tim10` is not added yet. 
You can leave the remaining fix to me.

Reply:
I don't really understand where `systick_rv4` comes from.
And `time-driver-systick` does not exist so the condition will always be true, right?

Reply:
> And time-driver-systick does not exist so the condition will always be true, right?

Yes. I'll add the required cfg flags in `build.rs`.

systick_rv4 comes from the following(generated via `build.rs`):

- https://github.com/ch32-rs/ch32-data/blob/main/data/registers/systick_rv4.yaml

And

https://github.com/ch32-rs/ch32-hal/blob/56de16c1aab4250d9beda0aeeb9b14712035a24c/build.rs#L76-L82